cmd/cli/plugin/package/README.md

```diff
@@ -8,46 +8,65 @@ Package and Repository Operations ( Subject to Change ):
 
 ```sh
 >>> tanzu package --help
-Tanzu package management
+Tanzu package management (available, install INSTALLED_PACKAGE_NAME --package-name PACKAGE_NAME --version VERSION, installed, repository)
 
 Usage:
+  tanzu package [flags]
   tanzu package [command]
```

Comment:
Does the first line means [flags] can be used without any explicit [command]? If not please merge into a single line to avoid confusion

Reply:
There is the `--help` flag available for all the commands, but I am not sure if we want to consider that here.